src/sphinx_tags/__init__.py

```diff
@@ -3,6 +3,7 @@
 """
 import os
 import re
+from collections import defaultdict
```

Comment:
Unused imports: this and `get_matching_files`. Those could be removed in your other PR, though.